content/blog/2015-07_Meet Nukomeeter  Adam.md

```diff
@@ -0,0 +1,40 @@
+---
+created_at: 2015-07-02
+kind: article
+publish: true
+title: "Meet Nukomeeter: Adam"
+authors:
+- adam
+- kasia
+tags: interview
```

Comment:
@myviridis  I've got compilation error here , this is wrong format for tags. It looks like authors above